lms/product/canvas/_plugin/grouping.py

```diff
@@ -78,6 +78,24 @@ def get_groups_for_grading(
             self._custom_course_id(course), grading_student_id, group_set_id
         )
 
+    def sections_enabled(self, request, application_instance, course):
+        params = request.params
```

Comment:
I know this would suck but I think if the interface for this was to only accept the `assignment` and then work out the rest (or inject it as a service initialization thing) it would go a long way to harmonising the interface with the groups.